StellarDotnetSdk/LedgerEntries/LedgerEntryContractCode.cs

```diff
@@ -6,14 +6,15 @@ namespace StellarDotnetSdk.LedgerEntries;
 
 public class LedgerEntryContractCode : LedgerEntry
 {
-    private LedgerEntryContractCode(byte[] hash, byte[] code, ExtensionPoint extensionPoint)
+    private LedgerEntryContractCode(byte[] hash, byte[] code, ExtensionPoint? extensionPoint, ContractCodeCostInputs? costInputs)
     {
         Hash = hash;
         Code = code;
         ExtensionPoint = extensionPoint;
+        CostInputs = costInputs;
     }
 
-    public ExtensionPoint ExtensionPoint { get; }
+    public ExtensionPoint? ExtensionPoint { get; }
```

Comment:
I think we solved this different before right? Can you make it more inline with the other version objects.

Reply:
This one is different. The others are not nullable, but this one is not.

Reply:
I meant more like the `AccountEntryExtensionV1` for example. You don't use a v1 class here while I do see a v1 in the xdr.

Reply:
Oh you're right. Fixed.